docs/installation/install-options/server-config.md

```diff
@@ -147,6 +147,7 @@ the agent options are there because the server has the agent process embedded wi
 |  `--cluster-dns` value   | "10.43.0.10"         | Cluster IP for coredns service. Should be in your service-cidr range |
 |  `--cluster-domain` value  | "cluster.local"        | Cluster Domain       | 
 |  `--flannel-backend` value   | "vxlan"      | One of 'none', 'vxlan', 'ipsec', 'host-gw', or 'wireguard'       |
+|  `----egress-selector-mode` value | "pod" | One of 'agent', 'cluster', 'pod', or 'disabled' |
```

Comment:
As a user, I'd honestly value having the explanations in the description of https://github.com/k3s-io/k3s/pull/5603, but not sure how this fits with your refactoring work on https://github.com/k3s-io/docs/pull/6